src/mgmt/rpc.h

```diff
@@ -70,10 +72,18 @@ enum mgmt_profile_op {
 	MGMT_PROFILE_STOP
 };
 
+/* clang-format off */
+
 extern struct crt_proto_format mgmt_proto_fmt_v3;
 extern struct crt_proto_format mgmt_proto_fmt_v2;
 extern int dc_mgmt_proto_version;
 
+#define DAOS_OSEQ_MGMT_OP /* output fields */                    \
+	((int32_t)		(mo_rc)			CRT_VAR) \
+	((struct rsvc_hint)	(mo_hint)		CRT_VAR)
```

Comment:
Do we need a uint32_t padding field between mo_rc and mo_hint?

Reply:
yes i think so. otherwise I recall we get some corruption in certain cases in the network stack. there are some examples in this file that show that padding being added. to have things 64 bit aligned.

Reply:
Looks like yes... Didn't think about this. I've updated this and the other new macros where appropriate.